src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline

```diff
@@ -147,6 +147,9 @@ public class LibraryServiceSettings extends ClientSettings {
       .build();
 
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
I didn't anticipate this before - this will have to be changed every time we bump up our version.

Let's wait until another day to fix it - we aren't frequently bumping up the version right now. I'm thinking maybe we need to support special directives in baselines for "match a dynamic value passed in" for when we do diffs.